neural_network/2_hidden_layers_neural_network.py

```diff
@@ -0,0 +1,203 @@
+import numpy
```

Comment:
Conventions say to import and use numpy as: `import numpy as np`

Reply:
I do not think that we need to enforce this.

Reply:
It's not about enforcing, it's about being consistent as I think all the scripts in this repository is using `numpy` like so.